python/rpdk/java/templates/generate/HandlerWrapper.java

```diff
@@ -108,8 +115,35 @@ public void testEntrypoint(
             response = ProgressEvent.defaultFailureHandler(e, HandlerErrorCode.InternalFailure);
         } finally {
             writeResponse(outputStream, response);
+            outputStream.close();
+        }
+    }
+    {% else %}
+    public static void main(String[] args) throws IOException {
+        if (args.length != 1){
+            System.exit(1);
+        }
+        final String outputFile = (UUID.randomUUID().toString() + ".txt");
```

Comment:
do we need to keep track of this output? or are we just getting the output from system out?

Reply:
No, we are just getting the output from system out